drivers/regex/mlx5/mlx5_regex.c

```diff
@@ -149,6 +151,8 @@ mlx5_regex_pci_probe(struct rte_pci_driver *pci_drv __rte_unused,
 		goto error;
 	}
 	priv->ctx = ctx;
+	/* Default RXP programming mode to Shared. */
```

Comment:
comment in the wrong format,

Reply:
Ok, but can you show me the format? I reviewed existing code and coding style link, but all show single line comments like I have written.